src/markdown/tables.scss

```diff
@@ -13,13 +11,12 @@
 
     th,
     td {
-      padding: 6px 13px;
-      border: 1px solid lighten($gray-300, 5%);
+      padding: ($spacer-2 - 2) ($spacer-3 - 2);
+      border: $border-width $border-style $border-gray-dark;
```

Comment:
Is `$border-gray-dark` the same thing as the lighten method or are we cool with the color change here?

Reply:
Here screenshots of the different border colors:

`$border-gray` | `lighten($gray-300, 5%)` | `$border-gray-dark`
--- | --- | ---
![border-gray](https://user-images.githubusercontent.com/378023/66192288-16655500-e6cb-11e9-9df2-31e063343b35.png) | ![lighten](https://user-images.githubusercontent.com/378023/66192287-16655500-e6cb-11e9-8da5-4fad37eddff6.png) | ![border-gray-dark](https://user-images.githubusercontent.com/378023/66192289-16655500-e6cb-11e9-85ba-404726a5a0cf.png)
`hsl(214, 13%, 90%)` | `hsl(210, 10%, 89%)` | `hsl(213, 11%, 84%)`

`$border-gray-dark` would make it a bit darker. `$border-gray` would make it a bit lighter, but the difference is not as big.

Hmmm.. I'm on the fence, but I slightly lean towards using `$border-gray`. If the border is not too strong, you can focus better on the content. Not sure if that's a valid argument. :grimacing: